akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala

```diff
@@ -298,7 +307,10 @@ object MediaTypes extends ObjectRegistry[(String, String), MediaType] {
   private def registerFileExtensions[T <: MediaType](mediaType: T): T = {
     mediaType.fileExtensions.foreach { ext ⇒
       val lcExt = ext.toLowerCase
-      require(!extensionMap.contains(lcExt), s"Extension '$ext' clash: media-types '${extensionMap(lcExt)}' and '$mediaType'")
+      /* Remove this condition of already registered media type should application/font-woff media type only once this media type is removed */
```

Comment:
Similar to https://github.com/akka/akka-http/pull/2320/files#diff-a3293e8a7bed4283fabbccebd3cb5800R333, I think we can remove the association of the `woff` file extension with the 'old' content type. That would be a change in behavior, but a desirable one AFAICS.